docs/spam.md

```diff
@@ -23,15 +23,37 @@ Another effective method of combating spam is to use reCAPTCHA. If you enable Ak
   <input type="hidden" name="options[reCaptcha][siteKey]" value="{SITE-KEY}">
   <input type="hidden" name="options[reCaptcha][secret]" value="{ENCRYPTED-SECRET}">
   ```
-1. Follow the reCAPTCHA documentation to add either a <a href="https://developers.google.com/recaptcha/docs/display#auto_render" class="cta">checkbox</a> or <a href="https://developers.google.com/recaptcha/docs/invisible#auto_render" class="cta">invisible challenge</a> to your form
+1. Add the reCAPTCHA credentials to your staticman.yml
+  ``` yml
+    reCaptcha:
+      enabled: true
+      # siteKey and secret should match your config.toml
+      siteKey: ""
+      # (!) ENCRYPT reCaptcha secret key using Staticman /encrypt endpoint
+      # i.e. https://{your-staticman-api}/v3/encrypt/{your-site-secret}
+      # For more information, visit https://staticman.net/docs/encryption
+      secret: ""
+  ```
```

Comment:
This is the same as the current step 2:
> 2. Use the Staticman encryption endpoint to encrypt the secret and add the reCAPTCHA options to your site config

Can move this example yaml up so it's right under that step?

Reply:
Also, which config is `config.toml` referring to?

Reply:
I believe `config.toml` a hugo-specific reference. I removed it. And the other comment about encrypting the secret was redundant with the text in step 2, so I removed that too.

I haven't implemented reCAPTCHA on my own jekyll/staticman sites, so I'm not sure if these instructions could be further clarified.